include/seccomp.h.in

```diff
@@ -198,44 +198,108 @@ struct scmp_arg_cmp {
  */
 #define SCMP_SYS(x)		(__NR_##x)
 
+/* Helper macros for the 32-bit argument comparison macros */
+/* Expand to the number of arguments given */
```

Comment:
You can probably just drop this comment.  Since we aren't intending this for use by callers, some "mystery" might be good :)